packages/solo/solo-config.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
I'm delighted to see the ag-solo get a smaller number of vats. That should reduce startup time too.